### ISSUE TITLE
RuntimeException: Failed to poll on ingestion status - when setting creationTime more than past 7 days (SparkIngestionProperties) inconsistent error

### ISSUE THREAD

**Issue by talsheldon** [Azure/azure-kusto-spark#193]
**Describe the bug**
When writing to Kusto, with `creationTime` more than 7 days in the `SparkIngestionProperties` I'm thrown with the exception `RuntimeException: Failed to poll on ingestion status.`.
The bug wasn't really consistent - I tried a couple of times with the time of last week and sometimes it worked and sometimes it didn't work.

**To Reproduce**
```
val conf: Map[String, String] = Map(
      KustoSourceOptions.KUSTO_AAD_APP_ID -> kustoAadAppId,
      KustoSourceOptions.KUSTO_AAD_APP_SECRET -> kustoAadAppSecret,
      KustoSourceOptions.KUSTO_AAD_AUTHORITY_ID -> kustoAadAuthorityID,
    )

val a = DateTime.now().plusDays(-7)
val sparkIngestionProperties = Some(new SparkIngestionProperties(creationTime=a))

df.write.kusto(kustoCluster, kustoDatabase, tenant, conf, sparkIngestionProperties)
```

Sometimes it didn't work with `-6` either. 

**Expected behavior**

The error:

```
RuntimeException: Failed to poll on ingestion status.
	at com.microsoft.kusto.spark.utils.KustoClient.$anonfun$finalizeIngestionWhenWorkersSucceeded$2(KustoClient.scala:287)
	at com.microsoft.kusto.spark.utils.KustoClient.$anonfun$finalizeIngestionWhenWorkersSucceeded$2$adapted(KustoClient.scala:249)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at com.microsoft.kusto.spark.utils.KustoClient.$anonfun$finalizeIngestionWhenWorkersSucceeded$1(KustoClient.scala:249)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
```

**Desktop (please complete the following information):**
Azure DBR 8.1 | Spark 3.1.1 | Scala 2.12
With library `com.microsoft.azure.kusto:kusto-spark_3.0_2.12:2.7.5`.

**Comment by daniel-dqsdatalabs** [comment]
I'm getting the same issue

**Comment by ohadbitt** [comment]
Hi 
we are working on a fix
This actually is a result of wrong deserialization of the result which in this case returns the next :

**Comment by ohadbitt** [comment]
Fixed on 2.8.0
